lib/apiservers/engine/backends/image.go

```diff
@@ -280,3 +272,29 @@ func imageConfigToDockerImageInspect(imageConfig *metadata.ImageConfig, productN
 
 	return inspectData
 }
+
+func clientFriendlyTags(imageName string, tags []string) []string {
```

Comment:
A comment about what this func is doing here would be useful for people who don't yet have the 2000 yard stare from digging through docker code. :)